src/ui/public/vis/vis.js

```diff
@@ -12,13 +12,12 @@ import _ from 'lodash';
 import AggTypesIndexProvider from 'ui/agg_types/index';
 import RegistryVisTypesProvider from 'ui/registry/vis_types';
 import VisAggConfigsProvider from 'ui/vis/agg_configs';
-import PersistedStateProvider from 'ui/persisted_state/persisted_state';
+import { PersistedState } from 'ui/persisted_state';
```

Comment:
This one is a little weird because its just using PersistedState for an instanceof comparison. While I agree it would probably be a little more accurate to use the angular version, I don't have the injector in here to grab it, and it works correctly as is because the angular version extends the base version.